katalyze/main.go

```diff
@@ -41,6 +41,8 @@ import (
 
 var (
 	outputDir       = flag.String("output_dir", "", "Directory for returning the processed SGFs. By default, uses current directory")
+	bucket          = flag.String("bucket", "", "The bucket for CloudStorage of analyzed games. This param is optional.")
```

Comment:
seems like you need a flag that's like `--storage-provider={localdisk|gcp}`

Reply:
Good point. I was considering that. That does seem nicer.